testinfra/modules/process.py

```diff
@@ -110,7 +110,9 @@ def _get_process_attribute_by_pid(self, pid, name):
 
     @classmethod
     def get_module_class(cls, host):
-        if (host.system_info.type == "linux"
+        if host.system_info.distribution == "alpine":
+            return BusyboxProcess
```

Comment:
Is it also possible to have procps installed in alpine ?
Maybe we should detect busybox with `if host.file('/bin/ps').linked_to == '/bin/busybox')` ?

Reply:
That is a much nicer method of detection of Busybox!  I will make that change.

It is possible to have "procps" installed on Alpine, but it isn't there by default I don't believe. 

The test image does currently have it installed (left around in my hacking about), which means  `BusyBoxProcess` is not being properly called or tested.  I will remove "procps" from the Alpine image and make any changes required.